templates/layout/mainnavigation.html

```diff
@@ -55,14 +55,22 @@
 {{ define "mainNavigationItem" }}
   {{ if not .IsHidden }}
     <a class="dropdown-item" href="{{ .Path }}">
```

Comment:
Try this to align the menu items:
```suggestion
   <a class="dropdown-item" {{ if .IsHighlighted }}style="padding:0 0.625rem;"{{ end }} href="{{ .Path }}">
```